.circleci/manifest-feature.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
@andrewoh531 I think these two manifest files and the shell script(s) should live at the root of the repo, or in a `ci` folder rather than in the `.circleci` folder. They're not really related to CircleCI so better off out of the folder (e.g. I could deploy locally using these same manifest files).

Reply:
Discussed offline - will move the scripts into a `ci` folder